message/infrastructure/io/doc.go

```diff
@@ -0,0 +1,3 @@
+package io
+
+// io transforms the standard library interfaces io.Writer and io.Reader into Publisher and Subscriber, respectively.
```

Comment:
it will be nice to add link to docs

Reply:
cef2cc9795955fd7984e8080c918fd4c2c66a009 done

ofc, the link will become active after merge